### DISCUSSION TITLE
acreditación de mojo

### DISCUSSION THREAD

**Discussion by bramase** [Chia-Network/chia-blockchain#8521] [Support]
Tengo el inconveniente de que actualice el software de Chia para unirme a la pool oficial, solicite un mojo pero nunca se me acredito. Ya actualice el programa, me encuentro sincronizado y me aparece el mensaje de que mi solicitud ya fue enviada pero nuca se me acredito en mi billetera.

**Comment by loppefaaret** [comment]
verify that both your "Full Node" and "Wallet" are both fully synced!
otherwise, there are more faucets out there that are willing to send you some mojos, if you just google "chia faucet"

**Comment by BrandtH22** [comment]
@bramase , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
